alphalens/performance.py

```diff
@@ -137,7 +138,7 @@ def factor_returns(factor_data, long_short=True, group_neutral=False):
         asset belongs to.
     long_short : bool
         Should this computation happen on a long short portfolio? if so, then factor values
-        will be demeaned across factor universe when factor weighting the portfolio. 
```

Comment:
Trailing whitespaces have been removed as an artifact of my text editor, which automatically removes them. This may cause conflicts that need to be resolved once #179 is merged.